MapboxSearch/offline/src/androidTest/java/com/mapbox/search/offline/OfflineSearchEngineIntegrationTest.kt

```diff
@@ -525,51 +523,31 @@ internal class OfflineSearchEngineIntegrationTest {
                 createCoreSearchAddress(
                     houseNumber = "2011",
                     street = "15th Street Northwest",
-                    place = "Washington",
-                    // TODO (SWEB-1113)
-                    // region = "District of Columbia",
+                    // See SWEB-1113 for more history
+                    region = createCoreSearchAddressRegion("District of Columbia")
                 )
             ),
-            // TODO (SWEB-1113)
-            // descriptionAddress = "2011 15th Street Northwest, Washington, District of Columbia",
-            descriptionAddress = "2011 15th Street Northwest, Washington",
+            // See SWEB-1113 for more history
+            descriptionAddress = "2011 15th Street Northwest, District of Columbia",
 
             distanceMeters = 0.0,
-            center = Point.fromLngLat(-77.03402549505554, 38.91792475903431),
+            center = Point.fromLngLat(-77.03404491238354, 38.9179071535832),
             serverIndex = null,
         )
 
-        const val DOUBLE_COMPARISON_EPS = 0.00001
-
-        fun Double.approximatelyEquals(other: Double) = abs(this - other) < DOUBLE_COMPARISON_EPS
+        fun assertSearchResultEquals(expected: OfflineSearchResult, actual: OfflineSearchResult): Boolean {
 
-        fun Point?.approximatelyEquals(other: Point?): Boolean {
-            if (this == other) return true
-            if (this == null || other == null) return false
-            return latitude().approximatelyEquals(other.latitude()) && longitude().approximatelyEquals(other.longitude())
-        }
+            assertEquals(expected.name, actual.name)
+            assertEquals(expected.descriptionText, actual.descriptionText)
+            assertEquals(expected.address, actual.address)
+            assertEquals(expected.coordinate, actual.coordinate)
```

Comment:
We already had issues with comparing coordinates and at least at that time it was expected that coordinates might be slightly different due to tiles building process. @globaltrouble is this still actual?

I think we should compare double approximately anyway

Reply:
@DzmitryFomchyn yes, you are right, we need to compare it with some delta, because address can be updated and it's coordinate can be changed (+/- 5-10 m)

Reply:
Done